docs/source/user/generating_users.rst

```diff
@@ -25,10 +25,14 @@ This command generates users whose usernames start with a particular prefix. Usa
 
 Note that the expiration and password-length are optional parameters which default to never and 8 respectively.
 
+In the response, you can find the field ``user_credentials`` containing the list of users created
+(example: [['username', 'password'], ['sample_user', 'BBuOb5sN']]") and field ``pdf_link``
+containing a download link to a PDF file containing user credentials.
+
 Adding from admin inteface
 --------------------------
 
-At the url ``/admin/openwisp_radius/radiusbatch/add`` one can directly generate users using the prefix and the number of users. A PDF can be downloaded immediately after the users have been generated.
+At the url ``/admin/openwisp_radius/radiusbatch/add`` one can directly generate users using the prefix and the number of users. A PDF containing the list of user credentials can be downloaded from the admin interface button:
```

Comment:
I am not convinced of this passage because we mention the download button which is only shown at the end of the gif for a brief moment.
Can you add two screenshots (using png which should be lighter), one which shows just the download button and another one that shows the generated PDF?

Then, before the gif we could add something like:

`Here's a sequence of images that show how this feature works:` or something similar.

This would make the passage more complete and easy to understand at glance.